packages/cli/src/cog/tms.util.ts

```diff
@@ -8,22 +8,14 @@ export interface LatLon {
 
 export const TmsUtil = {
     /** Convert a tile to a BBox in Wsg84 units */
-    tileToLatLon(tms: TileMatrixSet, tile: Tile): LatLon {
-        const { forward } = getProjection(tms.projection, Epsg.Wgs84);
-        const point = tms.tileToSource(tile);
-        const [lon, lat] = forward([point.x, point.y]);
-        return { lat, lon };
-    },
-
-    /** Convert a tile to a BBox in Wsg84 units */
-    tileToWsg84Bounds(tms: TileMatrixSet, tile: Tile): Bounds {
-        const ul = this.tileToLatLon(tms, tile);
-        const lr = this.tileToLatLon(tms, { x: tile.x + 1, y: tile.y + 1, z: tile.z });
-        return new Bounds(ul.lon, lr.lat, Math.abs(lr.lon - ul.lon), Math.abs(lr.lat - ul.lat));
+    tileToSourceBounds(tms: TileMatrixSet, tile: Tile): Bounds {
+        const ul = tms.tileToSource(tile);
+        const lr = tms.tileToSource({ x: tile.x + 1, y: tile.y + 1, z: tile.z });
+        return new Bounds(ul.x, lr.y, Math.abs(lr.x - ul.x), Math.abs(lr.y - ul.y));
     },
 
     /**
-     * Convert a tile to a GeoJson Polygon in Wsg84 units
+     * Convert a tile to a GeoJson Polygon in Source units
      */
     tileToPolygon(tms: TileMatrixSet, tile: Tile): [number, number][][] {
```

Comment:
is this `[number, number][][]` a geojson type ?

Reply:
`Position[][]`